### PR TITLE
[Excel] (Com to Web) Updates doc to note new host support (ppt, word) 

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ec8dfa9](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/ec8dfa98975058f566b6ff8c7b7dfa17df62241f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906110026028282-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7LboBDyy1qLQm65FnmyyI0yzKn4XpU97CtQfDYfIRVg%3D&st=2019-06-11T00%3A21%3A56Z&se=2019-07-12T00%3A26%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
I'm leaving this in the Excel category for now but we could also rearrange this so info on com to web is in a more general area. Also, did we have a spot to put info on newly released features and/or a changelog? (I just don't remember off the top of my head) because this strikes me as a candidate for any kind of list like that.

File: docs/excel/xll-compatibility-requirements.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 ---
 # COM add-in and XLL UDF compatibility requirements (preview)
 
-COM add-in and XLL UDF compatibility is currently available on the following platforms:
+COM add-in compatibility is currently available on the following platforms:
 
 - Excel Online
 - Excel on Windows (64-bit version 1810 or later)
+- Excel for Mac
+- Word and Powerpoint (Note that for these two products, you must have an Office 365 subscription and join the [Office Insider](https://products.office.com/office-insider) program at the **Insider** level -- formerly called "Insider Fast")
```
Comment (mscharlock) [inline docs/excel/xll-compatibility-requirements.md]:
```suggestion
- Word and Powerpoint (Note that for these two products, you must have an Office 365 subscription, join the [Office Insider](https://products.office.com/office-insider) program at the **Insider** level -- formerly called "Insider Fast", and be using version 16.0.11629.20214 of Word or Powerpoint).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ed8df41](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/ed8df41161ba287591ed186a936979a66656c2c7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906110033375890-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RJxoDpBN%2FUEz0GF8ApywWlN%2BLjrARLIflxVoQKLwhY4%3D&st=2019-06-11T00%3A29%3A33Z&se=2019-07-12T00%3A34%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@mscharlock @davidchesnut Keyur stopped by a minute ago to chat about this; I told him I'd get this changed published today. (I've added myself as reviewer and will merge the PR and publish to live whenever this change is complete, as it's already past due for publication.) Thanks!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b46d33b](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/b46d33b584dc1678bd842e2d062de62c3eb1203d)_: 

### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906131931495775-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nek9Nsn8rO79d0Hdx3jlsUjRMYqDwgU6JeM9hON0%2FBU%3D&st=2019-06-13T19%3A27%3A54Z&se=2019-07-14T19%3A32%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[773482f](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/773482feb15878fc281ee88d254109fd4a6863bc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132047180662-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IaCmNo3VMc53YJwi0Tiw2nrNg568ar8sMqt2lZRvqaY%3D&st=2019-06-13T20%3A43%3A16Z&se=2019-07-14T20%3A48%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0cf84ab](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/0cf84aba1379dab98f668902d5ff40010621a09e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132053168240-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CEzeMXRWJ4tWuG2aJK2WiQvPmZey%2BOtXRwpSofv6rzk%3D&st=2019-06-13T20%3A49%3A13Z&se=2019-07-14T20%3A54%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6778fca](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/6778fca17e338d48f91e8ec872df1dfcb74e3ba7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132100069039-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=N9cTvQDhbJPN7pQRAaze9zOb%2FJbU9BxxOveZqC37S2E%3D&st=2019-06-13T20%3A56%3A10Z&se=2019-07-14T21%3A01%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[538beff](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/538beffec9a00f22e92933e922df6db1609a16a5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132116379445-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kpxRfs99h2oQ3amsLrzg3PvF7EhYUkL5oVsmkAZgCtA%3D&st=2019-06-13T21%3A12%3A32Z&se=2019-07-14T21%3A17%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[42ceb15](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/42ceb15f44c31cb078930de60d725da412587436)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132127140526-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rfmz6%2BMWMeF8ileX0nv82kE31ofLet4L2Xk%2BIcVwtrE%3D&st=2019-06-13T21%3A23%3A11Z&se=2019-07-14T21%3A28%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf8ce7b](https://github.com/OfficeDev/office-js-docs-pr/pull/1070/commits/bf8ce7b4ee49916d50efc0059a1bfd2322083161)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-comtoweb/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132130014767-1070%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qax2nKI%2Ff4F7AZIH65Acruv8QLVRQXB%2BuxJ3uMGoF%2Fo%3D&st=2019-06-13T21%3A26%3A01Z&se=2019-07-14T21%3A31%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**